packages/ui/src/pages/MyStudiesPage/components/MyStudiesItem.vue

```diff
@@ -14,7 +13,48 @@ const { mutate } = useMutation({
 </script>
 
 <template>
-  <div class="flex border-b border-neutral-300 py-6 items-center gap-6">
+  <RouterLink
+    :to="{ name: 'study', params: { id } }"
+    class="block hover:bg-neutral-50 transition-colors"
+  >
+    <div
```

Comment:
There's a lot of divs here, so I'm wondering if it could be simplified after switching to using an HTML or Element Plus table to list all the studies. Maybe we could get rid of MyStudiesItem and change it to MyStudiesTable so all studies are contained in one component